libaums/src/main/java/com/github/mjdev/libaums/driver/scsi/ScsiBlockDevice.kt

```diff
@@ -257,15 +302,15 @@ class ScsiBlockDevice(private val usbCommunication: UsbCommunication, private va
      */
     @Synchronized
     @Throws(IOException::class)
-    override fun read(devOffset: Long, dest: ByteBuffer) {
+    override fun read(deviceOffset: Long, buffer: ByteBuffer) {
         //long time = System.currentTimeMillis();
-        require(dest.remaining() % blockSize == 0) { "dest.remaining() must be multiple of blockSize!" }
+        require(buffer.remaining() % blockSize == 0) { "buffer.remaining() must be multiple of blockSize!" }
 
-        readCommand.init(devOffset.toInt(), dest.remaining(), blockSize)
+        readCommand.init(deviceOffset.toInt(), buffer.remaining(), blockSize)
         //Log.d(TAG, "reading: " + read);
 
-        transferCommand(readCommand, dest)
-        dest.position(dest.limit())
+        transferCommand(readCommand, buffer)
```

Comment:
Shouldn't `NotReadyTryAgain` or `InitRequired` caught and handled here? Same for `write()`.